docs/guide-de-CH/README.md

```diff
@@ -0,0 +1,29 @@
+Twitter Bootstrap Erweiterung für Yii 2
```

Comment:
I think it should be `guide-de`. German in Switzerland should be OK for other German-speaking countries, right? If so, it's better to join efforts.

Reply:
Of course it's nearby the same. There is just one (little) difference the germans have to the swiss (and that's why I named it de-CH): [the ß char](https://en.wikipedia.org/wiki/%C3%9F)
But that's no reason they wont understand 😄 . I'll refactor it.